core/src/main/scala/org/apache/spark/deploy/security/HadoopDelegationTokenManager.scala

```diff
@@ -230,6 +230,8 @@ private[spark] class HadoopDelegationTokenManager(
         val now = System.currentTimeMillis
         val ratio = sparkConf.get(CREDENTIALS_RENEWAL_INTERVAL_RATIO)
         val delay = (ratio * (nextRenewal - now)).toLong
+        logInfo(s"Calculated delay on renewal is $delay, based on next renewal $nextRenewal " +
+          s"and the ratio $ratio")
```

Comment:
Although the log message has timestamp, it would be good to have `now` information together in addition to `nextRenewal` in this *log* line because it will be subtracted and can be *minus*.